src/main/java/io/strimzi/kafka/bridge/mqtt/Main.java

```diff
@@ -0,0 +1,23 @@
+package io.strimzi.kafka.bridge.mqtt;
+
+import io.netty.channel.ChannelOption;
+import io.netty.channel.nio.NioEventLoopGroup;
+import io.strimzi.kafka.bridge.mqtt.core.MqttServer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class Main {
+    private static final Logger logger = LoggerFactory.getLogger(Main.class);
+    static final int PORT = Integer.parseInt(System.getProperty("port", "1883"));
+
+    public static void main(String[] args) {
+        try {
+            NioEventLoopGroup masterGroup = new NioEventLoopGroup();
+            NioEventLoopGroup workerGroup = new NioEventLoopGroup();
+            MqttServer server = new MqttServer(PORT, masterGroup, workerGroup, ChannelOption.SO_KEEPALIVE);
+            server.start();
+        } catch (Exception e) {
```

Comment:
which exception this is supposed to catch? You are calling just `start` on the mqtt server and it's already catching exception and stopping the server if needed.